airbyte-webapp/src/components/ui/NextTable/NextTable.module.scss

```diff
@@ -18,23 +23,48 @@
 
 // --------- <th/> ---------
 .th {
-  background: colors.$grey-50;
+  background: colors.$dark-blue;
   color: colors.$grey;
   text-transform: uppercase;
   border-bottom: none;
   text-align: left;
   font-size: 10px;
   line-height: 12px;
   font-weight: 600;
+  width: auto;
+  padding: 9px 13px 10px;
+
+  &.light {
+    background: none;
+    text-transform: capitalize;
+    border-bottom: none;
+    text-align: left;
+    font-size: 11px;
+    line-height: 12px;
+    font-weight: 400;
+  }
+
+  &.collapse {
+    /* stylelint-disable-next-line number-max-precision */
+    width: 0.0000000001%;
+  }
```

Comment:
Just curious, this sounds to be like a row collapse.
Is this style just set minimum width?